Product/Production/Services/PatientDiscoveryCore/src/test/java/gov/hhs/fha/nhinc/patientdiscovery/inbound/deferred/request/StandardInboundPatientDiscoveryDeferredRequestTest.java

```diff
@@ -79,36 +82,31 @@ public void standardInboundPatientDiscoveryRequest() {
         MCCIIN000002UV01 expectedResponse = new MCCIIN000002UV01();
 
         PatientDiscoveryPolicyChecker policyChecker = mock(PatientDiscoveryPolicyChecker.class);
-        AdapterPatientDiscoveryDeferredReqErrorProxyObjectFactory proxyErrorFactory = mock(AdapterPatientDiscoveryDeferredReqErrorProxyObjectFactory.class);
-        PatientDiscoveryAuditLogger auditLogger = mock(PatientDiscoveryAuditLogger.class);
-        AdapterPatientDiscoveryDeferredReqProxyObjectFactory adapterFactory = mock(AdapterPatientDiscoveryDeferredReqProxyObjectFactory.class);
+        AdapterPatientDiscoveryDeferredReqErrorProxyObjectFactory proxyErrorFactory
+            = mock(AdapterPatientDiscoveryDeferredReqErrorProxyObjectFactory.class);
+        PatientDiscoveryDeferredRequestAuditLogger auditLogger = mock(PatientDiscoveryDeferredRequestAuditLogger.class);
```

Comment:
PatientDiscoveryDeferredRequestAuditLogger is used in several methods. Can it final object when the class get loaded?